sdk/core/src/context.rs

```diff
@@ -70,14 +128,34 @@ impl Context {
             .flatten()
     }
 
-    /// Returns the number of entities in the type map.
+    // Returns the number of entities present in the type map. An overridden entity counts as one
+    // regardless on how many times it has been overridden.
     pub fn len(&self) -> usize {
```

Comment:
Do we actually need this method?

Reply:
No, I don't think we do. 
I just added it because it was fun to write :neckbeard: ...

I'll remove it.